tools/workspace-plugin/src/plugins/workspace-plugin.ts

```diff
@@ -22,34 +22,73 @@ import { assertProjectExists, projectConfigGlob } from './shared';
 import { buildCleanTarget } from './clean-plugin';
 import { buildFormatTarget } from './format-plugin';
 import { buildTypeCheckTarget } from './type-check-plugin';
+import { measureStart, measureEnd } from '../utils';
 
-interface WorkspacePluginOptions {}
+interface WorkspacePluginOptions {
+  testSSR?: TargetPluginOption;
+  verifyPackaging?: TargetPluginOption;
+}
+interface TargetPluginOption {
+  targetName?: string;
+  /**
+   * project names to exclude from adding target
+   */
+  exclude?: string[];
+  /**
+   * project names to include for adding target
+   */
+  include?: string[];
+}
 
 export const createNodesV2: CreateNodesV2<WorkspacePluginOptions> = [
   projectConfigGlob,
-  (configFiles, options, context) => {
-    return createNodesFromFiles(
+  async (configFiles, options, context) => {
+    const globalConfig: Pick<TaskBuilderConfig, 'pmc'> = { pmc: getPackageManagerCommand('yarn') };
```

Comment:
`getPackageManagerCommand` is very expensive ( that was the main cause of perf issues ) - now it's gonna be called only once instead of N times